### ISSUE TITLE
pug  <3.0.1 - preview-email  <=2.0.2 - Severity: high

### ISSUE THREAD

**Issue by Beloin** [nest-modules/mailer#526]
Npm audix cannot fix High dependency security problems.

```
node_modules/pug
  @nestjs-modules/mailer  *
  Depends on vulnerable versions of preview-email
  Depends on vulnerable versions of pug
  node_modules/@nestjs-modules/mailer
  preview-email  <=2.0.2
  Depends on vulnerable versions of pug
  node_modules/preview-email

3 high severity vulnerabilities
```

This is fixable by updating this packages, or do we need a mailer update?

**Comment by travenin** [comment]
Depebdabot has opened a PR for this: https://github.com/nest-modules/mailer/pull/522

**Comment by Beloin** [comment]
Node has found 2 high severity vulnerabilities. With Dependencies of @nestjs-modules/mailer.
Output of ```npm audit```:
```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ High          │ Remote Code Execution                                        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ pug                                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=3.0.1                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ @nestjs-modules/mailer                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ @nestjs-modules/mailer > preview-email > pug                 │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/1643                            │
└───────────────┴──────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ High          │ Remote Code Execution                                        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ pug                                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=3.0.1                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ @nestjs-modules/mailer                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ @nestjs-modules/mailer > pug                                 │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/1643                            │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

Are this only complaining because are out of date? If not, are some others problems involving those third-party packages?
